libraries/lib-graphics/FrameStatistics.cpp

```diff
@@ -0,0 +1,130 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
Is this a comment convention we should do in all files?

There should be more regularity but I'm not sure what exactly it should be.

Reply:
This is a standardized approach, so yes, I would like it see more of it!